samples/bluetooth/mesh/light_ctrl/src/main.c

```diff
@@ -21,7 +21,7 @@
 #include <mpsl/mpsl_lib.h>
 #endif
 
-#define EMDS_DEV_IRQ 26
+#define EMDS_DEV_IRQ 29
```

Comment:
I'm curious to know if this change works for all platforms and why did we choose 26 then?

Reply:
It works on nRF52840, nRF5340 and nRF54L15. I don't know where 26 came from. 29 was found by iterating through interrupt lines in the running sample and listing the free and implemented ones. (As far as I understand, the 26 came from a similar process)

Reply:
It was set to 26 here https://github.com/nrfconnect/sdk-nrf/commit/89b50afd9eefb4966e94f75c04ccd1098d421779

@m-alperen-sener : As far as I understand, it was a similar case of "find a different, available interrupt number" that I had to do here?

Reply:
How about nrf52832/nrf52833? They are in the list of supported platforms too

Reply:
In the sample? I don't think we claim the actual sample code to be a stable API?

Reply:
AFAIK we use the interrupt that is free to use; there is no background to that unfortunately.

Reply:
Now I have also tested the interrupt on nRF52832 and nRF52833, it works. Will remove the DNM now.

Reply:
Instead of hardcoding the interrupt number, you could use the defines from the MDK.
For nRF52840/nRF52832/nRF52833 29 is the PDM_IRQn.
For the nRF5340, 29 is EGU2_IRQn.
For nRF54L15 29 seems not to be a valid irq actually.
Maybe e.g. Software interrupt 1 would be better choice.

Reply:
@sigurdnev I was not aware of these defines from the MDK. I find `SWI01_IRQn = 29` in nrf54l15_application.h, so it seems to be SW interrupt 1 already? How do I use these MDK headers in the application? Do I need to conditionally include the correct one based on platform or is there a single header I can include that will magically include the correct defines for the current platform?

Reply:
I'll update this to not use hardcoded IRQn then. DNM again for now.